collector/test/CollectorConfigTest.cpp

```diff
@@ -145,4 +151,64 @@ TEST(CollectorConfigTest, TestEnableExternalIpsRuntimeConfig) {
   EXPECT_TRUE(config.EnableExternalIPs());
 }
 
+TEST(CollectorConfigTest, TestYamlConfigToConfigMultiple) {
+  std::vector<std::tuple<std::string, bool, bool>> tests = {
```

Comment:
Having 2 bools in the vector of tests does not make it obvious that one is for the expected value and the other is for checking if the yaml is valid or not. Split invalid yaml to its own test and keep this one for checking the value of valid yaml configuration. This should also simplify the tests overall.

Reply:
I also now see there is just one case for invalid yaml, so splitting it to its own makes even more sense to me.

Reply:
Done